src/main_app/db/__init__.py

```
@@ -2,28 +2,52 @@
 
 logger = logging.getLogger(__name__)
 
 
+def _enable_sqlite_foreign_keys(dbapi_connection, connection_record):
+    """Enable foreign key enforcement for SQLite connections."""
+    try:
+        cursor = dbapi_connection.cursor()
+        cursor.execute("PRAGMA foreign_keys = ON")
+        cursor.close()
+    except Exception:
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)